Added Timer

FelixMueller10 opened the pull request:
I hope it works

FelixMueller10 committed:
Style geändert

FelixMueller10 committed:
Created new file for testing, probably useless

FelixMueller10 committed:
insertet Timer that counts up for how long you
have benn playing

FelixMueller10 committed:
Merge remote-tracking branch 'upstream/main'

Joaveinz commented:
Thanks! 
I see the new functionality is in index3.html

Can we have it part of the main index.html?

@FelixMueller10

FelixMueller10 committed:
changed font size of timer

FelixMueller10 commented:
The timer was actually already part of index.html but it was just really small, so I changed the font-size. Index3.html was just a file in which I tested some stuff for myself, and I deleted it now.
